model_example_controller:
    type: icra18_controllers/ModelExampleController
    arm_id: panda

plug_in_controller:
    type: icra18_controllers/PlugInController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
